modzy/models.py

```diff
@@ -29,6 +31,129 @@ def __init__(self, api_client):
         self._api_client = api_client
         self.logger = logging.getLogger(__name__)
 
+    def get_model_details(self, model, version):
+        """
+        Checks to see if a model with a certain id and version is active, and if it is, will return the model
+        details for that particular model.
+
+        Args:
+            model: model id, or `Model` instance
+            version: semantic version of previously specified model
+
+        Returns:
+            model: The model details for the model with the id and version specified or None if there are no active
+            models with these parameters
+
+        """
+        model_id = Model._coerce_identifier(model)
+
+        # TODO: this changed from being in the models api to the resources api,
+        #  so maybe it should go in a different module?
+        endpoint = "/resources/processing/models"
+
+        required_entitlements = "CAN_VIEW_MODELS_PROCESSING_STATE"
+        allowed = self._api_client.accounting.has_entitlement(required_entitlements)
```

Comment:
I guess that this validation is made on the server-side where, if you don't have the entitlement you'll get a 40X error in the next call. 

So I guess that this is unnecessary, otherwise we will have to do the same with the rest of api calls

Reply:
Ok, considering this weird inconsistency with the services is ok to keep in this way.